src/index.html

```diff
@@ -1,16 +1,42 @@
 <!doctype html>
-<html lang="en">
+<html lang="ru">
+
   <head>
```

Comment:
Remove blank lines between ALL parent and child tags
```suggestion
<html lang="ru">
  <head>
```